client/app/bundles/Attendance/components/ClassesScreen/ClassesScreen.jsx

```diff
@@ -28,12 +26,13 @@ class ClassesScreen extends BaseComponent {
 
   render() {
     const { data, actions, location, params } = this.props;
-    let classes = data.get('$$classes');
+    let attendance = data.get('$$attendance');
+    console.log(attendance.get('classes'));
```

Comment:
Log statement - should the be committed?

Reply:
Oops, nice catch.